bin/test-merges.js

```diff
@@ -43,14 +43,22 @@ const TREEHERDER_PREFIX = "https://treeherder.mozilla.org/#/jobs?repo=pine&revis
 //
 // mozilla-central - the hg repo for firefox. Will be created if it doesn't
 // already exist.
-const WORKING_DIR = process.env.AS_PINE_TEST_DIR;
+const AS_PINE_TEST_DIR = process.env.AS_PINE_TEST_DIR;
+
+const TESTING_LOCAL_MC = path.join(AS_PINE_TEST_DIR, "mozilla-central");
 
 const SimpleGit = require("simple-git");
-const TESTING_GIT_AS = path.join(WORKING_DIR, AS_REPO_NAME);
-const git = new SimpleGit(TESTING_GIT_AS);
+const TESTING_LOCAL_GIT = path.join(AS_PINE_TEST_DIR, AS_REPO_NAME);
+const git = new SimpleGit(TESTING_LOCAL_GIT);
+
+// Mostly useful to specify during development of the test automation so that
+// prepare-mochitests-dev and friends from the development repo get used
+// instead of from the testing repo, which won't have had any changes checked in
+// just yet.
+const AS_PMD_GIT_DIR = process.env.AS_PMD_GIT_DIR || TESTING_LOCAL_GIT;
```

Comment:
Nice. I was needing to copy over files and stash them away to avoid them from preventing changes by `git`.